packages/passport-interface/src/api/requestEncryptedStorage.ts

```diff
@@ -0,0 +1,33 @@
+import { EncryptedPacket } from "@pcd/passport-crypto";
+import { LoadE2EERequest, LoadE2EEResponse } from "../RequestTypes";
+import { APIResult } from "./apiResult";
+import { POST } from "./constants";
+
+/**
+ * Never rejects. All information encoded in the resolved response.
+ */
+export async function requestEncryptedStorage(
+  passportServer: string,
+  blobKey: string
+): Promise<EncryptedStorageResult> {
+  const request: LoadE2EERequest = { blobKey };
+  const url = `${passportServer}/sync/load`;
+
+  try {
```

Comment:
Comment on the general pattern here: I like breaking out request logic into this interface package.  Is there an opportunity to break out more common logic into generic/shared code?  It strikes me that each request function in this directory is doing a mix of some request-specific logic (e.g. filling in specific request fields, or constructing/proving PCDs in some cases I saw elsewhere), and generic logic (like building a POST, JSON.stringify, checking errors).  It would be nice to be able to separate out and clearly see what's request-specific and what always happens for any request.

Reply:
lmk what you think of it now